conda/recipes/jupyterlab-nvdashboard/meta.yaml

```diff
@@ -1,7 +1,10 @@
+{% set package_json = load_file_data("package.json") %}
+{% set pyproject_toml = load_file_data("pyproject.toml") %}
```

Comment:
In the Conda recipe we can also load from `package.json`. So we do so here

We fallback to `pyproject.toml` for anything else